src/System.Collections.Concurrent/ref/System.Collections.Concurrent.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyVersion>4.0.10.0</AssemblyVersion>
```

Comment:
I was as much asking because I'm not confident I bumped the right places on something I'm working on :(